project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -139,6 +145,45 @@ public String getAllQuestionsJSON(String key) {
     return new Gson().toJson("");
   }
 
+/** 
+ * this function takes the assignment Key and answer group key as parameters
+ * returns list of blobKeys to submissions that fit in specified answer group for specific question
+ */ 
+  public List blobkeysFromAnswerGroup(String assignKey, String groupKey) {
```

Comment:
Why does this function return a list and not a json object?

Reply:
its a list of blobkeys (strings)
I think you're right -- it would make more sense to return a json of submissionkey->blobKey (?)

Reply:
yeah i think that would be better for javascript code

Reply:
Implemented.